lib/services/elasticsearch.js

```diff
@@ -1265,21 +1312,21 @@ class ElasticSearch extends Service {
     }
 
     return this._mExecute(esRequest, toImport, rejected)
-      .then(response => {
-        const results = [];
+      .then((response) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS3524)